.github/workflows/docker_doublet-detection.yml

```
@@ -0,0 +1,63 @@
+name: Build Docker image for doublet-detection
+
+concurrency:
+  # only one run per branch at a time
+  group: "doublet-detection-${{ github.ref }}"
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - analyses/doublet-detection/Dockerfile
+      - analyses/doublet-detection/.dockerignore
+      - analyses/doublet-detection/renv.lock
+      - analyses/doublet-detection/conda-lock.yml
+  push:
+    tags:
+      - "v*.*.*"
```

Tag pushes should be handled by the all-in-one workflow now.
```suggestion
    tags:
      - "v*.*.*"
```

---

For some reason, I don't have the `push:` block in the template workflow, but I think it probably should be here.